packages/roosterjs-content-model/lib/modelToDom/handlers/handleEntity.ts

```diff
@@ -40,14 +46,32 @@ export const handleEntity: ContentModelHandler<ContentModelEntity> = (
 
     if (!entity) {
         parent.appendChild(wrapper);
+
+        addDelimiterElements(wrapper, isReadonly, context);
     } else {
         // Create a comment as placeholder and insert into DOM tree.
         // If the entity DOM can be reused, the original DOM node will be preserved without any change
         // so that in case there is something that is sensitive to its DOM path (e.g. IFRAME), no need to cause it reloaded.
         // For entity that is not directly under root, later we will replace the comment with its original DOM node
-        parent.appendChild(createEntityPlaceholder(entity));
+        const placeholder = createEntityPlaceholder(entity);
+        parent.appendChild(placeholder);
+
+        addDelimiterElements(placeholder, isReadonly, context);
 
         // Save the entity DOM wrapper node and its placeholder into context so that later we know how to handle it
         context.entities[entity.id] = wrapper;
     }
 };
+function addDelimiterElements(
+    wrapper: HTMLElement,
+    isReadonly: boolean,
+    context: ModelToDomContext
+) {
+    if (
+        context.isFeatureEnabled?.(ExperimentalFeatures.InlineEntityReadOnlyDelimiters) &&
```

Comment:
For the content model code, I'd like it can be independent from editor code. So for this check, I prefer to use a simple variable inside `EditorContext` rather than a isFeatureEnabled() function:
```ts
interface EditorContext{
  ...
  addDelimiterForEntity: boolean
}
```

When create this context we can read its value from isFeatureEnabled() in ContentModelEditor. Later we just need to read it from EditorContext.

Reply:
Remove isFeatureEnabled and added the new property